common/app/model/meta.scala

```diff
@@ -257,7 +257,7 @@ final case class MetaData (
 
   val hasSlimHeader: Boolean =
     contentWithSlimHeader ||
-      sectionId == "identity" ||
+      (sectionId == "identity" && !id.startsWith("/user/")) ||
```

Comment:
Oh gosh. It isn't a nice way is it? Not sure how else we can do this though :woman_shrugging:

Reply:
we could remove the whole line really 😅 The rest of identity pages should be using `identityHeader` anyway.

Reply:
You do what you think is best :). I think you know better than me what the identity pages need!

Reply:
I think every other significant page is ported over but I wanna be more sure (so, wait s bit) before making the change!